src/calendar/index.tsx

```diff
@@ -46,9 +46,9 @@ export interface CalendarProps extends CalendarHeaderProps, DayProps {
   /** Always show six weeks on each month (only when hideExtraDays = false) */
   showSixWeeks?: boolean;
   /** Handler which gets executed on day press */
-  onDayPress?: (date: DateData) => void;
+  onDayPress?: (date?: DateData) => void;
```

Comment:
No no. You can't just change the return type. This is not part of this PR scope and it's a breaking change